bessctl/test_sugar.py

```diff
@@ -0,0 +1,100 @@
+import fnmatch
+import os
+import sugar
+import unittest
+
+
+class TestSugar(unittest.TestCase):
+    """
+    All scripts in conf/ will be dynamically added here as individual
```

Comment:
Is this the part that codecov detects? Presumably codecov won't detect the calls out to the shell? (How does the codecov integration work here?)